respect the libdir cmake already set up for us instead of using the multiarch crap

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -184,12 +184,6 @@ set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 # Installation directories
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    # Debian multiarch support
-    execute_process(COMMAND dpkg-architecture -qDEB_HOST_MULTIARCH
-        OUTPUT_VARIABLE ARCH_SUBDIR
-        ERROR_QUIET
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
     CHECK_INCLUDE_FILES(proc/readproc.h READPROC_H_FOUND)
     if (READPROC_H_FOUND)
         add_definitions (-DHAVE_READPROC_H)
@@ -205,11 +205,11 @@ if (WIN32 OR APPLE)
     set (LIB_ARCH_INSTALL_DIR lib)
 else ()
     set (DOC_INSTALL_DIR share/doc/${CMAKE_PROJECT_NAME})
-    set (LIB_INSTALL_DIR lib/${CMAKE_PROJECT_NAME})
+    set (LIB_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME})
     if (ARCH_SUBDIR)
-        set (LIB_ARCH_INSTALL_DIR lib/${ARCH_SUBDIR}/${CMAKE_PROJECT_NAME})
+        set (LIB_ARCH_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/${ARCH_SUBDIR}/${CMAKE_PROJECT_NAME})
     else ()
-        set (LIB_ARCH_INSTALL_DIR lib/${CMAKE_PROJECT_NAME})
+        set (LIB_ARCH_INSTALL_DIR ${LIB_INSTALL_DIR})
     endif ()
 endif ()
 
